crates/adapters/Cargo.toml

```diff
@@ -136,23 +136,23 @@ jemalloc_pprof = "0.1.0"
 psutil = "3.2.2"
 
 [dev-dependencies]
-actix-test = "0.1.1"
-bstr = { version = "0.2.1", features = ["serde1"] }
-serde_json = "1.0.127"
+actix-test = { workspace = true }
+bstr = { workspace = true, features = ["serde1"] }
+serde_json = { workspace = true }
 size-of = { version = "0.1.5", package = "feldera-size-of", features = ["time-std", "ordered-float"] }
```

Comment:
Why is this package excluded? I would especially expect this to be workspace-wide declared to keep track of when we have to modify packages.